namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/elasticache.tf

```diff
@@ -35,5 +35,9 @@ resource "kubernetes_secret" "elasticache" {
     auth_token               = module.elasticache.auth_token
     redis_url                = "rediss://:${module.elasticache.auth_token}@${module.elasticache.primary_endpoint_address}:6379"
   }
+
```

Comment:
Please remove the tags as they are supported only for aws resources

Reply:
is this not a resource? I am looking to get read access to elasticache in particular